handlers/handlers.go

```diff
@@ -0,0 +1,197 @@
+package handlers
+
+import (
+	"database/sql"
+	"encoding/json"
+	"log"
+	"net/http"
+	"strconv"
+
+	"github.com/codingpop/simplerest/db"
+	"github.com/go-chi/chi"
+)
+
+// Handlers hold all the route handlers
+type Handlers struct {
+	db *db.DB
```

Comment:
Advanced Go exercise: turn this into an [interface](https://gobyexample.com/interfaces). You would also update the parameter in `New` to the same interface type. This removes the dependency of this package on `github.com/codingpop/simplerest/db`, which means it's easier to change out and easier to test.